### ISSUE TITLE
First yanked element is displayed twice on Telescope picker

### ISSUE THREAD

**Issue by becarepsol** [AckslD/nvim-neoclip.lua#26]
When using Telescope to pick yank registers the first element that I yanked is
displayed twice, as if I had yanked twice.

Subsequent yanked elements behave correctly, meaning that they don't get
duplicated

This is my current config:

```lua
	use({
		"AckslD/nvim-neoclip.lua",
		event = "TextYankPost",
		config = function()
			require("neoclip").setup({
				enable_persistant_history = true,
				keys = {
					i = {
						paste = "<c-y>",
						paste_behind = "<c-b>",
					},
				},
			})
		end,
		requires = {
			"tami5/sqlite.lua",
			module = "sqlite",
		},
	})
```

![neoclip](https://user-images.githubusercontent.com/90428811/136131327-dc034220-57a2-4a16-8461-d46e68eac80c.png)

Thanks

**Comment by AckslD** [comment]
Hi @becarepsol! Thanks for reporting this.

It does not happen for me, do you perhaps have some local config interacting with `TextYankPost`? 

Would you mind for example adding a print statement [here](https://github.com/AckslD/nvim-neoclip.lua/blob/ecf149d4baab347a101d427c3d472730504182e8/lua/neoclip/handlers.lua#L30) to see if for some reason the handler of the yank is called twice or if there is another issue?

**Comment by becarepsol** [comment]
Hi @AckslD Thanks for the reply,

The only other config that interacts with `TextYankPost` is this helper
function to easily define autogroups

```lua
local funcs = {}

function funcs.define_augroups(definitions)
	for group_name, definition in pairs(definitions) do
		vim.cmd("augroup " .. group_name)
		vim.cmd("autocmd!")

		for _, def in pairs(definition) do
			local command = table.concat(vim.tbl_flatten({ "autocmd", def }), " ")
			vim.cmd(command)
		end

		vim.cmd("augroup END")
	end
end

return funcs
```

That then gets call to setup `vim.highlight.on_yank()`

```lua

local func = require("utils.augroup")

func.define_augroups({
	_general_settings = {
		{ "TextYankPost", "*", "lua require('vim.highlight').on_yank({higroup = 'YankHighlight', timeout = 200})" },
	},
})
```

And yes, after removing `event = 'TextYankPost'` in the `packer` config it
works as expected, no first yank duplication

Finally I just want to further clarify that this unexpected behavior only
happens with the first yanked element after freshly starting vim, and further
yanks are added only once to the clipboard as expected

Thanks

**Comment by AckslD** [comment]
@becarepsol This seems to be a bug in packer which I could reproduce, I opened a small PR https://github.com/wbthomason/packer.nvim/pull/636 which I think should fix this.
